django/contrib/gis/geos/factory.py

```diff
@@ -15,12 +17,10 @@ def fromfile(file_h):
 
     # If we get WKB need to wrap in memoryview(), so run through regexes.
     if isinstance(buf, bytes):
-        try:
+        with suppress(UnicodeDecodeError):
             decoded = buf.decode()
```

Comment:
Not certain, but it seems like maybe only this line should be in the context manager.